docs/setting-up-local-development.md

```diff
@@ -76,6 +76,10 @@ These instructions assume you're using a package manager for your OS:
   $ sudo apt-get install rbenv ruby-build
   $ rbenv install X.Y.Z
   ```
+  If you are having trouble getting the version required, clone ruby-build via git instead of apt-get:
```

Comment:
Was there a useful error message? Which command raised it? rbenv? 
Might be more helpful than just 'having trouble'